subprojects/docs/src/docs/release/notes.md

```diff
@@ -34,6 +34,24 @@ For Java, Groovy, Kotlin and Android compatibility, see the [full compatibility
 
 <!-- Do not add breaking changes or deprecations here! Add them to the upgrade guide instead. -->
 
+## File system watching is ready for production use
+
+In an [incremental build](userguide/more_about_tasks.html#sec:up_to_date_checks), input and output files are checked to determine what needs to be rebuilt.
+This feature typically saves a lot of time; however, it adds some I/O overhead, which can be noticeable in large projects when not much has changed since the previous build. 
+
+Back in Gradle 6.5 we've introduced _[file-system watching](userguide/gradle_daemon.html#sec:daemon_watch_fs)_ as an experimental feature.
+When enabled, it allows Gradle to keep what it has learned about the file-system in memory during and between builds instead of polling the file system on each build.
+This significantly reduces the amount of disk I/O needed to determine what has changed since the previous build.
+
+This feature is now ready for production use and supported on Linux, Windows and macOS.
+You can enable it by supplying the parameter `--watch-fs` on the command-line.
```

Comment:
Should folks enable the feature by adding `org.gradle.vfs.watch` to their `gradle.properties`? The command line flag won't help much, since they need to have at least two builds to benefit from file system watching.

Reply:
Updated.

Reply:
I think mentioning one way how to enable the feature is enough. And I would choose the `gradle.properties` way. Adding both options is too much for the release notes in my opinion.

Reply:
I'm not sure how you want to do that exactly. Can you make the changes you think are necessary?

Reply:
I pushed what I had in mind.